Actions/CheckForNewWarnings/CheckForWarningsUtils.psm1

```diff
@@ -0,0 +1,90 @@
+<#
+    .SYNOPSIS
+    Ensures that a specified directory exists. If the directory does not exist, it creates it.
+    .DESCRIPTION
+    Ensures that a specified directory exists. If the directory does not exist, it creates it.
+#>
+function Initialize-Directory
```

Comment:
Let's move this one to AL-Go-Helper.ps1 for now.

Reply:
Skipping for now, waiting for refactoring.